django/dispatch/dispatcher.py

```diff
@@ -64,10 +66,6 @@ def connect(self, receiver, sender=None, weak=True, dispatch_uid=None):
 
                 Receivers must be able to accept keyword arguments.
 
-                If receivers have a dispatch_uid attribute, the receiver will
```

Comment:
Okay, I see what you mean. I think the comment is meant to be "If receivers are connected with a dispatch_uid argument, the receiver will not be added if another receiver was already connected with that dispatch_uid." Does that make sense?